requirements.txt

```diff
@@ -31,17 +31,17 @@ gevent==1.4.0
 greenlet==0.4.15
 gunicorn==19.9.0
 idna==2.7
-ipython==4.1.2
+ipython==7.34.0
```

Comment:
Are all of these used by the application code? Some like ipython might be from developer machines and not used?